src/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -1238,11 +1238,18 @@ static smt_termt convert_expr_to_smt(
 
 static smt_termt convert_expr_to_smt(
   const object_size_exprt &object_size,
-  const sub_expression_mapt &converted)
+  const sub_expression_mapt &converted,
+  const smt_object_sizet::make_applicationt &call_object_size)
```

Comment:
Why is this named `call_object_size`? Shouldn't this be called `object_size` or am I missing something?